examples/build.gradle

```diff
@@ -3,7 +3,16 @@ plugins {
     id "com.github.johnrengelman.shadow" apply false
 }
 
-println("Building against OpenTelemetry version: ${project.properties["io.opentelemetry.version"]}")
+ext {
```

Comment:
Moved the ext block outside the subprojects block since subprojects will fall back to their parent project's ext properties and this makes it possible to print the version here.